src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -440,6 +444,15 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 		return cell
 	}
 
+	private func familyTestCell(_ unseenCount: Int) -> UITableViewCell {
```

Comment:
I haven't checked it out yet, but I think it makes sense to add this cell to the preloaded cells after line 128 to prevent animation glitches on initial scrolling and only configure it once using a property holding the cell as we did for the other self-updating cells. (Will only make sense after you update the cell internally as we do for the others)